tools/build-tool/code/tooltexturegeneration.pas

```diff
@@ -421,34 +454,36 @@ procedure AutoGenerateTextures(const Project: TCastleProject);
 
         tcETC1:             PVRTexTool(InputFile, OutputFile, C, 'ETC1');
                       // or Compressonator(InputFile, OutputFile, C, 'ETC_RGB');
-        tcASTC_4x4_RGBA:           PVRTexTool(InputFile, OutputFile, C, 'ASTC_4x4');
```

Comment:
For some time, I'd say to keep the PVRTexTool option (and thus, DDS format for ASTC compressed textures) possible, under `{$ifdef CASTLE_ASTC_PVR_DDS}` symbol (documented in `castleconf.conf`).

While I think that new solution is better (`.astc` file format is simpler than `.dds`, `astcenc` is open-source), but for such change I'd say to keep the old version existing. We may want to sometimes test whether it results in the same things, with the same quality, speed. (even if PVRTexTool just calls astcenc for this job -- but then we want to be able to verify this by actually running it e.g. on Unholy/Escape data.)

Reply:
PVRTexTool is buggy in this case - and this specific call works only with new version. It would fail to process all the US assets (requiring both height and width be divisors of the NxM), just throws an exception and quits. There isn't much use in keeping it.

Reply:
OK, agreed.